src/components/Profile/components/SchedulePanel/components/ScheduleDialog/index.jsx

```diff
@@ -38,8 +37,11 @@ const ScheduleDialog = (props) => {
             <>
               <add-to-calendar-button
                 name={props.selectedCourseInfo.title}
-                startDate={format(new Date(props.selectedCourseInfo.start), 'yyyy-MM-dd')}
-                //endDate={format(new Date(props.selectedCourseInfo.end), 'yyyy-MM-dd')}
+                startDate={format(
+                  setDay(new Date(props.selectedCourseInfo.start), 4),
```

Comment:
```suggestion
                  setDay(new Date(props.selectedCourseInfo.start), 4),
```
why are we still hard-coding 4?

Reply:
We haven't pushed the commit with the changes yet, I will make sure to change this before review